docs/source/jupyter/sdk.ipynb

```diff
@@ -67,15 +78,29 @@
     }
    ],
    "source": [
-    "from typing import List, Dict\n",
-    "from marketplace.app import MarketPlaceApp\n",
-    "\n",
-    "mp = MarketPlaceApp(client_id=\"<registered application's client_id>\",\n",
-    "                    access_token=\"<your access token>\")\n",
-    "\n",
+    "from marketplace.app import get_app\r\n",
+    "\r\n",
+    "mp = get_app(app_id=\"<registered application's client_id>\")\r\n",
+    "\r\n",
     "print(mp.heartbeat())"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The token and host can also be passed to the get_app function as arguments:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mp = get_app(app_id=\"<registered application's client_id>\", access_token=\"<your access token>\")"
```

Comment:
I don't think we should promote passing the `access_token` directly, users should be encouraged to pass a custom client as introduced with https://github.com/materials-marketplace/python-sdk/pull/35.

Reply:
Okay I'll remove this example altogether